app/components/appcomponents/AdministratorCompo.js

```
@@ -17,32 +17,38 @@ const AdministratorCompo = () => {
+              <div>
+
+                <h3 className="text-[24px] text-[#3C3E41] font-variation-customOpt24 font-semibold text-center mt-[23.5px] tablet:mt-[30px]">
+                  Mesečna svečka
+                </h3>
+                <div
+                  style={{
+                    textShadow: "0px 3px 4px #00000050",
+                    color: "#2D3D48",
+                    fontWeight: 400,
+                    fontSize: "32px",
+                    fontVariationSettings: "'opsz' 32",
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)